fastlane/docs/Actions.md

```diff
@@ -1592,6 +1592,15 @@ git_branch
 ```
 
 
+### git_tag_exists
+
+Checks whether the given tag exists in the repo
+
+```ruby
+git_tag_exists(tag: "1.1.0")
```

Comment:
The code sample should probably show how to use this value, how about
```ruby
if git_tag_exists(tag: "1.1.0")
  UI.message("🚀")
end
```
This way people who are not familiar with Ruby know how to properly use it 👍